src/pages/practice/innersource.md

```diff
@@ -0,0 +1,56 @@
+---
+templateKey: practice-page
+title: InnerSource
+subtitle: Re-use code, collaborate, and use open practices inside your organisation.
+date: 2024-05-31T08:11:26.394Z
+authors:
+  - kattekrab
+  - rmarting
+tags:
+  - culture
+mobiusTag: foundation
+icon: /images/innersource-for-dummies.jpeg
+whatIs: I﻿nnerSource is an approach to software development inside an
+  organisation that is inspired by the success of the Open Source Software
+  movement.  A diverse community of practitioners is sharing and refining
+  [patterns at InnerSource commons](https://patterns.innersourcecommons.org/)
+whyDo: I﻿nnerSource promotes collaboration, and code-reuse. People say it's a
+  useful way to share knowledge, improve code quality, and increase the reach
+  and impact of their work.
```

Comment:
I would like to extend the why with some comment such as "opening silos" ... something similar to:

`InnerSource promotes collaboration, code-reuse, opening organisational silos to share freely code, and useful content along the organisation".

Also we could add the following bullets about Why InnerSource?

* Accelerate development and encourage collaboration
* Increase transparency and accountability
* Enhance employee engagement

Reply:
Yep! Good suggestion. Will add.

Reply:
updating now like this:

> I﻿nnerSource promotes collaboration, and code-reuse. It opens organisational silos, and facilitates the flow of useful content to those who need it.
> 
> People say it's a useful way to share knowledge, improve code quality, and increase the reach and impact of their work. 
> 
> * Accelerate development and encourage collaboration
> * Increase transparency and accountability
> * Enhance employee engagement